lib/database.js

```diff
@@ -4,6 +4,15 @@ const { Pool } = require('pg');
 
 const OPERATORS = ['>=', '<=', '<>', '>', '<'];
 
+const formatWhere = (where) =>
+  where
+    .filter((condition) => !!condition)
+    .map((condition) =>
+      Object.entries(condition).filter((entry) => entry[1] !== undefined)
+    )
+    .filter((params) => params.length !== 0)
+    .map(Object.fromEntries);
```

Comment:
We don't need this, at least in this PR, please describe a purpose of this code on separate issue